# youzan standard words (youzan-standard-words)

## Rule Details

This rule aims to...

Examples of **incorrect** code for this rule:

```js
const a = '登陆';
```

Examples of **correct** code for this rule:

```js
const a = '登录';
```

