# youzan standard words (youzan-standard-words)

## Rule Details

This rule aims to...

Examples of **incorrect** code for this rule:

```js
const a = '登陆';
```

Examples of **correct** code for this rule:

```js
const a = '登录';
```

## Link

[有赞标准词汇库](https://design.youzan.com/strategy/standard-library.html)
